core/server/src/main/java/alluxio/master/FaultTolerantAlluxioMaster.java

```diff
@@ -98,11 +97,13 @@ public void start() throws Exception {
 
           // When transitioning from master to standby, recreate the masters with a read-only
```

Comment:
I think one nice thing with re-creating was that no left over state was possible.

Also, please update this comment that it is not "recreated".